lib/src/transactions.dart

```diff
@@ -82,7 +82,7 @@ class ArweaveTransactionsApi {
 
     while (!uploader.isComplete) {
       if (!dryRun) {
```

Comment:
Is this just an infinite loop when doing dry runs 🤔

Reply:
I think so yes. It was just something I added before to test upload preparation only so it didn't matter at the time.